### PR TITLE
feat: Added routes and all pages ui

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
Added :
Router and added UI for all the Pages .

### CODE REVIEW

File: src/Stylesheets/navbar.css [hunk 1]
```diff
@@ -1,38 +1,64 @@
...
     justify-content: flex-start;
     width: 100%;
+    height: 4rem;
+    background: #181818;
+    border: none;
+    color: var(--white);
+    outline: none;
+    border-radius: 2px;
+    padding: 1rem;
+
```
Comment (Avin008) [inline src/Stylesheets/navbar.css:50]: remove blank spaces

File: src/components/Context/GetContext.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const [video, setVideo] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const { data } = await axios.get("/api/videos");
+        console.log(data.videos);
+        setVideo(data.videos);
+      } catch (error) {
+        console.log(error);
```
Comment (Avin008) [inline src/components/Context/GetContext.jsx:16]: remove console.log

File: src/components/Explore Page/ExplorePage.css [hunk 3]
```diff
@@ -3,28 +3,84 @@
...
 
 .sidebar {
-    background-color: #353535;
-    height: 93vh;
+    background-color: #202020;
+    height: 90vh;
     flex: .15;
-    border-right: 2px solid red;
+    /* border-right: 2px solid red; */
+
```
Comment (Avin008) [inline src/components/Explore Page/ExplorePage.css:10]: remove black spaces

**Review by Avin008 — APPROVED** [review]
looks good to merge

File: src/App.js [hunk 4]
```diff
@@ -3,12 +3,26 @@ import { BiSearchAlt } from "react-icons/bi";
 import LandingPage from "./Pages/LandingPage";
 import { Routes, Route } from "react-router-dom";
 import ExplorePage from "./Pages/ExplorePage";
+
+import Navbar from "./components/Navbar";
+
+import HistoryPage from "./Pages/HistoryPage";
+import WatchLaterPage from "./Pages/WatchLaterPage";
+import PlaylistPage from "./Pages/PlaylistPage";
+import LikePage from "./Pages/LikePage";
```
Comment (tusharkandpal) [inline src/App.js:12]: can use barrel export.

File: src/App.js [hunk 5]
```diff
@@ -3,12 +3,26 @@ import { BiSearchAlt } from "react-icons/bi";
...
+      {/* <LandingPage /> */}
       <Routes>
         <Route path="/" element={<LandingPage />} />
-        <Route path="explore" element={<ExplorePage />} />
+        <Route path="/explore" element={<ExplorePage />} />
+        <Route path="/history" element={<HistoryPage />} />
+        <Route path="/like" element={<LikePage />} />
+        <Route path="/watchlater" element={<WatchLaterPage />} />
+        <Route path="/playlist" element={<PlaylistPage />} />
       </Routes>
```
Comment (tusharkandpal) [inline src/App.js:26]: suggestion: can give a separate `WrapperFunction` for all these routes.

File: src/components/Context/GetContext.jsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+const Info = createContext();
+const InfoProvider = ({ children }) => {
+  const [video, setVideo] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const { data } = await axios.get("/api/videos");
+        console.log(data.videos);
```
Comment (tusharkandpal) [inline src/components/Context/GetContext.jsx:13]: please remove this logger.

File: src/components/Explore Page/ExplorePage.css [hunk 7]
```diff
@@ -3,28 +3,84 @@
...
+
+
+
 }
 
 .sidebar__component {
     height: 5rem;
-    border: 2px solid rebeccapurple;
+    /* border: 2px solid rebeccapurple; */
```
Comment (tusharkandpal) [inline src/components/Explore Page/ExplorePage.css:18]: remove this commented css please

File: src/components/Explore Page/ExplorePage.css [hunk 8]
```diff
@@ -3,28 +3,84 @@
...
+.main__content {
+    height: 90vh;
+    background-color: var(--light-dark);
+    ;
     flex: .85;
+    overflow-y: scroll;
+}
+
+.badges__mainContent {
+    /* border: 2px solid orange; */
```
Comment (tusharkandpal) [inline src/components/Explore Page/ExplorePage.css:44]: here too

File: src/components/Explore Page/ExplorePage.css [hunk 9]
```diff
@@ -3,28 +3,84 @@
...
+    height: 5rem;
+    display: flex;
+    padding: 6px;
+
+}
+
+.badges__mainContent button {
+    color: var(--white);
+    background-color: #2a2727;
+    /* #2a2727 */
```
Comment (tusharkandpal) [inline src/components/Explore Page/ExplorePage.css:56]: +1

File: src/components/VideoPage/videos.css [hunk 10]
```diff
@@ -0,0 +1,120 @@
+.video__section {
+    color: white;
+    /* border: 2px solid red; */
```
Comment (tusharkandpal) [inline src/components/VideoPage/videos.css:3]: +1

File: src/components/VideoPage/videos.css [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+    height: inherit;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.video__body {
+    /* border: 2px solid green; */
```
Comment (tusharkandpal) [inline src/components/VideoPage/videos.css:10]: +1

File: src/components/VideoPage/videos.css [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+    background-color: #574d4d;
+    width: 3rem;
+    display: grid;
+    place-items: center;
+    border-radius: 4px;
+    padding: 1px;
+}
+
+.video__details {
+    /* border: 2px solid yellow; */
```
Comment (tusharkandpal) [inline src/components/VideoPage/videos.css:42]: +1

**Review by tusharkandpal — APPROVED** [review]
Base structure looks good.!
hierarchy is well maintained & folder structure too.

**Pull request merged**